src/functions.jl

```diff
@@ -11,15 +11,15 @@ const PNG_COLOR_TYPE_GRAY_ALPHA = PNG_COLOR_MASK_ALPHA # 4
 const PNG_COLOR_TYPE_RGB_ALPHA = PNG_COLOR_MASK_COLOR | PNG_COLOR_MASK_ALPHA # 6
 const PNG_COLOR_TYPE_RGBA = PNG_COLOR_TYPE_RGB_ALPHA   # 6
 
-function png_error_handler(::Ptr{Void}, msg::Cstring)
+function png_error_handler(::Ptr{Nothing}, msg::Cstring)
```

Comment:
`Nothing` may be better spelled `Cvoid` here and in most other places in this PR. It's another name for the same thing, but probably clearer to talk in terms of the C types if you're using this as a callback.

Generally speaking, the type to use for most basic C types `blah` is the julia type `Cblah`... eg `void->Cvoid`, `size_t->Csize_t` etc.

Reply:
Done! Thank you for your feedback Chris! Much appreciated!

Reply:
You're welcome!